makedist

```diff
@@ -298,49 +303,66 @@ if $INCLUDEHELPER; then
 fi
 
 FUSESUBDIR=false
-CVMFSFUSE3URL="$CVMFSURL/`echo $CVMFSRPMNAME|sed 's/cvmfs-/cvmfs-fuse3-/'`"
+CVMFSFUSE3RPMNAME=`echo $CVMFSRPMNAME|sed 's/cvmfs-/cvmfs-fuse3-/'`
+CVMFSFUSE3URL="$CVMFSURL/$CVMFSFUSE3RPMNAME"
+
+USEFUSE3=false
 if $SING; then
+    USEFUSE3=true
+fi
+if $FUSE3; then
+    USEFUSE3=true
+fi
```

Comment:
I don't see a need to use a `$FUSE3` variable instead of using `$USEFUSE3` throughout.

Reply:
OK, I'll refactor this to use single variable.